package.json

```diff
@@ -23,7 +23,7 @@
         "build": "yarn clean && node scripts/build.js",
         "compile": "node scripts/compile.js",
         "clean": "rimraf dist && rimraf dist-es && rimraf dist-ts && rimraf node_modules/.cache",
-        "ci": "yarn lint && yarn prettier-check && yarn ts-check && yarn circular-dependencies && yarn build && yarn storybook-static && yarn test --ci && yarn test-acceptance --ci",
+        "ci": "yarn lint && yarn prettier-check && yarn ts-check && yarn circular-dependencies && yarn build && yarn storybook-static && yarn test --ci && cross-env PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true yarn test-acceptance --ci",
```

Comment:
please, can you set this env var in the gh workflow file instead?
I think it's the right place for this, and this way we won't have conflicts with my branch, where I've removed this `"ci"` from `package.json` script

Reply:
done